### DISCUSSION TITLE
extract data shared by another Garmin user

### DISCUSSION THREAD

**Discussion by chadwickyoung** [tcgoetz/GarminDB#244] [General]
Hey, I'm working on a research project with an undergraduate student.  She wants to gather Garmin data from the runners in a 5K and analyze that data. 

I see that GarminDB can be used to download your own data. Can you also use it to download data that has been shared with you?

For example, I have this link shared and made public on the Garmin connect App. Is it possible to download the data (in FIT, CSV, or other format)?

https://connect.garmin.com/modern/activity/16769408439?share_unique_id=11

Thanks,
Chad

**Comment by tcgoetz** [comment]
Not sure. May not work out of the box, but probably could be made to work. Have you tried it?
